app/queues/notifications.server.ts

```diff
@@ -44,7 +42,7 @@ notificationsQueue.on('error', (err) => {
  */
 export const notificationsWorker = new Worker<NotificationData>(
   'notifications',
-  async (job: Job) => {
```

Comment:
type assignment here overrode the `NotificationData` generic passed in when instantiating Worker and reverted job.data to have an explicit `any` type